examples/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-site/styles/components/text.less

```diff
@@ -50,7 +50,7 @@
 
             font-size: @cmp-examples-font-size;
             font-weight: @cmp-examples-font-weight-regular;
-            color: rgb(119, 119, 119);
+            color: rgb(80, 80, 80);
```

Comment:
Here we can use @cmp-examples-color-gray-700

Reply:
changed with https://github.com/adobe/aem-core-wcm-components/pull/483/commits/fb393fa8fd0ab3cccf88cd612d365e582d0c84e6